change/@microsoft-teams-js-52e4ec21-7735-45e1-882f-112e465ccc8d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This seems like it should be a "none" change since it isn't impacting the actual exported code in any way.